docs/helloserver/loadgen/loadgen.yaml

```diff
@@ -0,0 +1,65 @@
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START istio_helloserver_loadgen_deployment_loadgenerator]
```

Comment:
Consider renaming the region tags to be consistent with existing tags in this repo (e.g., starting with `servicemesh_`).
Feel free to ignore. Not a blocker.